xlb/operator/boundary_condition/bc_regularized.py

```diff
@@ -222,6 +216,15 @@ def functional_velocity(
             # Find normal vector
             normals = get_normal_vectors(missing_mask)
 
+            # Find the value of u from the missing directions
```

Comment:
We're not recomputing here we're simply reading it. I didn't notice any performance hit before and after the PR.

An improvement for this could be that we can store the missing direction and avoid the for loop for slightly improved performance. I'll try to do that.

Reply:
(this only works for linear/planar BCs like zouhe)